sources/osg/Matrix.js

```diff
@@ -652,9 +653,9 @@ var Matrix = {
     postMultRotate: ( function () {
         var r = matrixCreate();
         return function ( matrix, quat ) {
-            Matrix.makeRotateFromQuat( r, quat );
+            Matrix.makeRotateFromQuat( quat, r );
             Matrix.postMult( matrix, r );
-            return matrix;
+            return r;
```

Comment:
preMult( a, b ) -> a = a * b
postMult( b, a ) -> a = b * a
The "pre" and "post" simply tell where the result will be stored